python-threatexchange/threatexchange/api.py

```diff
@@ -511,3 +510,21 @@ def _postThreatDescriptor(self, url, postParams, showURLs, dryRun):
         except urllib.error.HTTPError as e:
             responseBody = json.loads(e.read().decode("utf-8"))
             return [None, e, responseBody]
+
+    def get_threat_descriptors_from_indicator(
+        self, indicator_id: int, showURLs: bool = False
+    ) -> t.List[t.Dict[str, t.Any]]:
+        url = (
```

Comment:
Would prefer [_get_graph_api_url](https://github.com/facebook/ThreatExchange/blob/d935e872ff89fd5dccc62f32951b17a0ffac3305/python-threatexchange/threatexchange/api.py#L335) used instead of handrolling URIs.

Reply:
For this and the 2 comments above I agree with you. Do you think its worth another PR to clean this up and bump the version again or save it for another time?